[Docs] Revise Vizro-MCP docs for Cursor (#1354)
mckinsey/vizro: vizro-mcp/docs/pages/tutorials/claude-desktop-tutorial.md

!!! Tip "PyCafe cannot display charts that use local data"

>     PyCafe will only open if your chart or dashboard uses data that can be downloaded from a public link. If your data is stored locally, you should copy the generated code into a `.py` file to run where it can access the data.

Reviewer: ```suggestion
    PyCafe will only open if your chart or dashboard uses data that can be downloaded from a public link. If your data is stored locally, you should copy the code generating the data set into a `.py` file to run.
```

Author: @huong-li-nguyen Not quite what I meant -- I was trying to say that you shouldn't use PyCafe with local data but instead create the code that Claude or Cursor generates into a .py and run it locally so it can access the data. 

Maybe I should say this:

```suggestion
Vizro-MCP can only show your chart or dashboard in PyCafe if the data it consumes can be downloaded from a public link. If you use local data, you should copy the generated chart or dashboard code into a `.py` file to run locally, so it can access the data.
```

Reviewer 2: While both is true I think we should go with your comment @stichbury because the above is nothing that the MCP server can do alone. So best ommit it I think
